golem/managers/proposal/plugins/blacklist.py

```diff
@@ -0,0 +1,22 @@
+from typing import Sequence
+
+from golem.managers import ProposalManagerPlugin
+from golem.resources import Proposal
+from golem.utils.logging import trace_span
+
+
+class BlacklistProviderIdPlugin(ProposalManagerPlugin):
+    def __init__(self, blacklist: Sequence[str]) -> None:
+        self._blacklist = blacklist
+
+    @trace_span(show_results=True)
+    async def get_proposal(self) -> Proposal:
+        while True:
+            proposal: Proposal = await self._get_proposal_callback()
```

Comment:
`self._get_proposal_callback()` is not a great name. It wrongly suggests that we will get a callback, not a proposal. `self._get_proposal` would be a better option?